test/Microsoft.Dnx.Testing.CommonUtils/DnxSdk/Dnu.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.Dnx.Testing
+{
+    public class Dnu
+    {
+        private readonly string _sdkPath;
```

Comment:
For now, just add the argument for additionalArguments for each method so the caller can specify additional arguments. I think this is what Victor meant.